pkg/probe/util_test.go

```diff
@@ -41,3 +42,103 @@ func TestFindPortByName(t *testing.T) {
 		t.Errorf("Expected %v, got %v, err: %v", want, got, err)
 	}
 }
+
+func TestResolveContainerPort(t *testing.T) {
```

Comment:
Can we run the test in parallel?
```suggestion
func TestResolveContainerPort(t *testing.T) {
	t.Parallel()
```

Reply:
Dup test name with https://github.com/kubernetes/kubernetes/pull/115708/files#diff-4a118db26fa2866f683a2cdf0f863edf63c9ac1b197e0c641b5ca6a3a669730bL46. 
Line 199 below